model/post.go

```diff
@@ -122,6 +122,12 @@ type PostForExport struct {
 	ReplyCount  int
 }
 
+type DirectPostForExport struct {
+	Post
+	User           string
```

Comment:
nit: Can we call this `Username` for clarity?

Reply:
We can, but using `User` aligns with what's in `DirectPostImportData`.  Should we keep it that way or try to make the use of `Username` consistent across the board?

Reply:
+1 for consistency -- fine as is :)